### DISCUSSION TITLE
TypeScript: serialize BigInt in JSON

### DISCUSSION THREAD

**Discussion by digerykevin** [community/community#126290] [Programming Help]
### Body

I'm looking for a way to force JSON.stringify to always print BigInts without complaining.

 if I try to print a JSON containing a BigInt, I get: 
`TypeError: Do not know how to serialize a BigInt.`

Any help is appreciated - many thanks in advance.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @digerykevin & @dev188007, thanks for being a part of the GitHub Community!

Please note: we made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As this discussion is almost an exact duplicate of the Stack Overflow conversation linked, I will be unmarking the answer and locking this, and other related posts.

Thanks for understanding.
